test/org.abapgit.adt.ui.test/src/org/abapgit/adt/ui/internal/wizards/TestPdeAbapGitRepositoriesSelectionWizard.java

```diff
@@ -0,0 +1,51 @@
+package org.abapgit.adt.ui.internal.wizards;
+
+import static org.easymock.EasyMock.*;
+import static org.junit.Assert.*;
+
+
+import org.abapgit.adt.backend.model.abapgitrepositories.IRepository;
+import org.abapgit.adt.ui.internal.wizards.AbapGitWizardSwitchBranch;
+import org.eclipse.core.resources.IProject;
+
+import org.junit.Before;
+import org.junit.Test;
+
+
+import com.sap.adt.tools.core.project.IAbapProject;
+
+public class TestPdeAbapGitRepositoriesSelectionWizard {
+
```

Comment:
- More test cases can be covered like invalid package or exception while fetching repo type.
- Also a UI test can be added to open the wizard, validate the fields in first page, then navigate to next page and validate if the branches are correctly set as input for combo viewer, and switch the branch in combo viewer. The Finish action can be skipped.